plugin/src/main/kotlin/me/gabber235/typewriter/Typewriter.kt

```diff
@@ -73,4 +76,20 @@ class Typewriter : KotlinPlugin() {
 		FactDatabase.shutdown()
 		adventure.close()
 	}
+
+	fun getCommandMap(): CommandMap {
+		var commandMap: CommandMap? = null
+
+		try {
+			val f: Field = Bukkit.getServer().javaClass.getDeclaredField("commandMap")
+			f.isAccessible = true
+			commandMap = f.get(Bukkit.getServer()) as CommandMap
+		} catch (e: NoSuchFieldException) {
+			e.printStackTrace()
+		} catch (e: IllegalAccessException) {
+			e.printStackTrace()
+		}
+
+		return commandMap?: throw IllegalStateException("Could not get command map")
+	}
```

Comment:
Please lmk if this function should be somewhere else.

Reply:
Yea I would put it somewhere in the utils folder